### ISSUE TITLE
Is it possible to get tweets by hashtags?

### ISSUE THREAD

**Issue by psanch21** [sns-sdks/python-twitter#133]
Hi,

I'm wondering if it is possible to use this package to return the most recent tweets that contain a given hashtag, or any other text, e.g., tweets containing WorldCup 2022.

Thanks a lot for your help in advance.

Best.

**Comment by MerleLiuKun** [comment]
Yes, you can use `search` endpoint to get tweets for your keywords
